sspl_test/sspl-test.spec

```diff
@@ -27,7 +27,7 @@ URL:        https://github.com/Seagate/cortx-sspl
 Source0:    %{name}-%{version}.tgz
 Requires:   %{product_family}-sspl = %{version}-%{release}
 BuildRoot:  %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
-Requires:  python36-psutil
+Requires:  python3-psutil
```

Comment:
This restricts the backward compatibility to centos 7.x or el7, since el7 has python36-psutil available and el8 has python3-psutil, I tried to find if it is possible to add something like this.
`Requires: python3-psutil or python36-psutil` but it didn't seem to work.
one workaround could be to validate in sspl_test phase and not to keep it as a build time dependency..

Reply:
Its referred as run time dependency, not build time

For pkg1 or pkg2 requirement, boolean dependencies are available in rpm version >= 4.13
https://rpm.org/user_doc/boolean_dependencies.html
https://serverfault.com/questions/299179/can-one-require-this-or-that-package-in-an-rpm-spec-file

Reply:
`Its referred as run time dependency, not build time`
yes sorry my bad, it validates at install time. correct.

`For pkg1 or pkg2 requirement, boolean dependencies are available in rpm version >= 4.13`
Thanks, @cdeshmukh. Addressed 👍 . I missed parenthesis earlier. 😅

Reply:
> I missed parenthesis earlier

yeah i saw :)